index.js

```diff
@@ -130,6 +158,9 @@ function Metric(namespace, units, defaultDimensions, options) {
   self._storedMetrics = [];
 
   if (self.options.enabled) {
+        if(!Object.keys(UNITS).find(u=>UNITS[u]===units)) {
+        throw 'cloudwatch-metrics: Unrecognised unit';
```

Comment:
nit: Can we use `Unrecognized` instead of `Unrecognised` (the second is the British spelling)?